### DISCUSSION TITLE
[HELP] Mount existing plots in Raspbian from remote UI (MacOS) and creating plots from remote UI.

### DISCUSSION THREAD

**Discussion by 2BLogic** [Chia-Network/chia-blockchain#3330] [Support]
Hi, 

Started to plot from Windows and switched to a Pi4 with Raspbian.

Setup : 
Raspbian on a Pi4 8G
Remote UI from MacOS
1 External HDD 6TB
1 External SSD 500GB (for temp files)

I already have some plots on my 6TB HDD.

how to mount existing plot into Raspbian to be seen on remote UI ? ( from command line ? )
How to create some plots remotely from MacOS UI since I can't "see" the drives connected to the Raspbian from MacOS.

Can anybody help me ? 

Thanks !

B.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
